src/components/SecondaryPanes/Frames/Frames.css

```diff
@@ -105,3 +104,7 @@
 :root.theme-dark .annotation-logo svg path {
   fill: var(--theme-highlight-blue);
 }
+
+.framework-component div {
```

Comment:
This selector is too general, as it would add 4px to every DIV within the panel.